perl-lib/OESS/lib/OESS/DB/Workgroup.pm

```diff
@@ -107,7 +107,7 @@ sub get_users_in_workgroup{
             next;
         }
         
-        push(@users, $user);
+        push(@users, $user->to_hash);
```

Comment:
Was this causing an issue elsewhere? A change like this will likely have far reaching effects.

Reply:
I didn't need this I changed it back and fixed another different issue I didn't notice with last names not being displated properly on the Workgroup page.